src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double base;
+    private final double height;
+
+    public RightTriangle(String color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (height * base) / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: right triangle,"
```

Comment:
Use 'name()' instead of 'toString()' when getting the String representation of enum constants. If 'getColor()' returns an enum, replace 'super.getColor()' with 'super.getColor().name()'.